modules/CbChargeSOMDriver/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+#
+# AUTO GENERATED - MARKED REGIONS WILL BE KEPT
+# template version 3
+#
+
+# module setup:
+#   - ${MODULE_NAME}: module name
+ev_setup_cpp_module()
+
+# ev@bcc62523-e22b-41d7-ba2f-825b493a3c97:v1
+# insert your custom targets and additional config variables here
+
+find_package(PalSigslot REQUIRED)
+
+add_subdirectory("libcbuart")
+add_subdirectory("chargesom")
+
+set(MODULE_DESCRIPTION ${PROJECT_DESCRIPTION})
+set(MODULE_VERSION ${PROJECT_VERSION})
```

Comment:
I disagree: we shouldn't start counting from zero for every new module, but we should show the project version because it will/should correspond to the tag in the git repo.